tcms/issuetracker/tests/test_kiwitcms.py

```diff
@@ -107,3 +108,10 @@ def test_report_issue_from_test_execution_1click_works(self):
             url=result['response'],
             is_defect=True,
         ).exists())
+
+    def test_empty_details_when_bug_dont_exist(self):
+        non_existing_bug_id = 1000
```

Comment:
It's better to use negative values in case there are so many objects created in the DB that suddenly make this a valid PK.